Project_team_1_HW2/Project_team_1_HW2.md

```diff
@@ -0,0 +1,35 @@
+
+To sum up, from the end-user's perspective, the situation our app is targeting is that of the multitude of news which is available to people in this day and age. Much like the above-mentioned APIs and the Netflix case, given the amount of news available on a daily basis to readers, it could be quite a daunting task to scour through countless websites to find news related to topics of interest, and this approach could turn out to be dispersive and time-consuming. By aggregating news content into a single up-to-date source and consolidating it into news bits displayed to the end user based on their preferences in content, our app hopes to simplify this task and "meet the user halfway" by proposing ready-to-access content they might be interested in.
+
+## 3. What are the key features of your system?
+
+Our system will have: 
+- Login feature - client can log in as a user.
+- API/RSS feature - to show news to the client of topics that they have chosen.
+- SQL database feature for user information - to keep user passwords, IDs, etc.
+- SQL database feature for news - to store latest news (topics, titles, etc.).
+
+## 4. Provide a sample architecture diagram of your system?
+
+![image](image.png)
+
+## 5. Describe a team approach to the project work?
+For us project team 1, we mainly used Slack, Atlassian, Google doc and GitHub as team collaboration tools. In the pre-group project stage (6/10-11/10), a meeting was held via zoom where each member introduced themselves and shared their areas of interest. After the discussion, we initially decided not to design a game, while most of the members were interested in the tech sector, we decided to start an APIs search and to think about what we would do based on the general direction of the tech industry. After that, we set up the Atlassian board to track each member’s progress and work goals, and decided to simplify the complexity of communication. Since slack comes with a huddle feature that not only supports multi-person calls and screen sharing, but also supports drawing in other people’s screen presentations, we decided to conduct the following meetings through slack huddle. During the start-up stage (12/10-16/10), we decided to make a software that consolidates information about the technology industry and makes it easy to find and read, and it should connect to SQL databases. In order to achieve that, we assigned different tech areas to different members to do APIs/RSS research and start writing simple code. At the same time we set up a Github repository to share our codes so that others can view or comment on it. We are still working on our respective parts of the code and will be discussing more about the database, login system and how to finish hw2 at the next meeting (19/10).
```

Comment:
4/6 well-described approached. However, the testing strategy is not mentioned in the approach.
Testing is one of the fundamental steps in software development